samples/python/com.ibm.streamsx.topology.pysamples/opt/python/streams/spl_samples.py

```diff
@@ -0,0 +1,140 @@
+# Licensed Materials - Property of IBM
+# Copyright IBM Corp. 2015
+
+# Simple inclusion of Python logic within an SPL application
+# as a SPL "Function" operator. A "Function" operator has
+# a single input port and single output port, a function
+# is called for every input tuple, and results in
+# no submission or a single tuple being submitted.
+
+# Import the SPL decorators
+from streamsx.spl import spl
+
+# Any function in a Python module (.py file) within the
+# toolkit's opt/python/streams directory is converted to a primitive operator
+# with a single input and output port. The primitive operator
+# is a C++ primitive that embeds the Python runtime.
+#
+# The function must be decorated with one of these
```

Comment:
namespace-info.spl/42 indicates that only functions decorated with @spl.pipe or @spl.sink are converted to primitive operators?  Hence, also, functions need not be decorated.